README.md

```diff
@@ -384,6 +384,7 @@ Services to securely store your Docker images.
 * [Docker Grand Ambassador](https://github.com/cpuguy83/docker-grand-ambassador) - This is a fully dynamic docker link ambassador. + [Article](https://docs.docker.com/engine/admin/ambassador_pattern_linking/) by [@cpuguy83][cpuguy83]
 * [docker-consul](https://github.com/gliderlabs/docker-consul) by [@progrium][progrium]
 * [etcd](https://github.com/coreos/etcd) - A highly-available key value store for shared configuration and service discovery by [@coreOS][coreos]
+* [istio](https://istio.io/) - An open platform to connect, manage, and secure microservices by [@IstioMesh](istio)
```

Comment:
GitHub link is preferred, as you did in the comment field.

Reply:
Thanks @devhkr it is now fixed